frontend/javascripts/admin/voxelytics/log_tab.tsx

```diff
@@ -192,6 +204,24 @@ export default function LogTab({
     }
   }, [logResult, showTimestamps]);
 
+  async function downloadFullLog() {
```

Comment:
I think this is fine for now, but we'll have to see whether this is feasible for very long logs. The alternative would be for the server to serve this log file in a streaming fashion, for direct download. Probably, voxelytics should log less if this becomes problematic, though :sweat_smile:

Reply:
💣